src/node_worker.cc

```diff
@@ -816,6 +816,68 @@ void Worker::Unref(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+void Worker::GetHeapStatistics(const FunctionCallbackInfo<Value>& args) {
+  Worker* w;
+  ASSIGN_OR_RETURN_UNWRAP(&w, args.This());
+
+  v8::HeapStatistics heap_stats;
+  w->isolate_->GetHeapStatistics(&heap_stats);
+
```

Comment:
If you use the object constructor that takes an array of keys and array of values, it will be faster

Reply:
Have you got an example I can copy from?

Reply:
Src/node_v8.cc line 350